specs/governance/alligator.md

```diff
@@ -0,0 +1,207 @@
+# Alligator
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Overview](#overview)
+- [Interface](#interface)
+  - [Core Functions](#core-functions)
+    - [`subdelegate`](#subdelegate)
+    - [`subdelegateFromToken`](#subdelegatefromtoken)
+    - [`subdelegateBatched`](#subdelegatebatched)
+    - [`afterTokenTransfer`](#aftertokentransfer)
+  - [Getters](#getters)
+    - [`getSubdelegations`](#getsubdelegations)
+    - [`getCheckpoints`](#getcheckpoints)
+    - [`getVotingPower`](#getvotingpower)
+- [Storage](#storage)
+- [Types](#types)
+  - [`SubdelegationRule`](#subdelegationrule)
+  - [`AllowanceType`](#allowancetype)
+  - [`Checkpoint`](#checkpoint)
+- [Backwards Compatibility](#backwards-compatibility)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Overview
+
+| Constant | Value                                        |
+|----------|----------------------------------------------|
+| Address  | `0x4200000000000000000000000000000000000043` |
+
+The `Alligator` contract implements subdelegations for the [`GovernanceToken`](gov-token.md). Subdelegations enable
+advanced delegation use cases, such as partial, time-constrained & block-based delegations, and relative & fixed allowances.
+
+The `Alligator` contract migrates the delegation state from the [`GovernanceToken`](gov-token.md) to itself
```

Comment:
I think that all of the backwards compat language should go in a "backwards compatibility" section rather than making it core to the spec of how the alligator contract works